R/CohortComparisonDiagnostics.R

```diff
@@ -18,75 +18,97 @@ computeCohortOverlap <- function(connectionDetails = NULL,
                                  connection = NULL,
                                  cohortDatabaseSchema,
                                  cohortTable = "cohort",
-                                 targetCohortId,
-                                 comparatorCohortId) {
+                                 targetCohortIds,
+                                 comparatorCohortIds,
+                                 batchSize = 200) {
   start <- Sys.time()
   
   if (is.null(connection)) {
     connection <- DatabaseConnector::connect(connectionDetails)
     on.exit(DatabaseConnector::disconnect(connection))
   }
   
-  if (!checkIfCohortInstantiated(
-    connection = connection,
-    cohortDatabaseSchema = cohortDatabaseSchema,
-    cohortTable = cohortTable,
-    cohortId = targetCohortId
-  )) {
-    warning(
-      "- Target cohort with ID ",
-      targetCohortId,
-      " appears to be empty. Was it instantiated? Skipping overlap computation."
-    )
-    delta <- Sys.time() - start
-    ParallelLogger::logInfo(paste(
-      "Computing overlap took",
-      signif(delta, 3),
-      attr(delta, "units")
-    ))
-    return(tidyr::tibble())
-  }
+  ## commenting out checks if cohort is instantiated, as this is not essential.
```

Comment:
Code should either be deleted or kept, we can let git manage the history :)

Reply:
good point!